### ISSUE TITLE
What is the plan for 0.20 to release?

### ISSUE THREAD

**Issue by yarikoptic** [python-versioneer/python-versioneer#251]
seeing current 0.19-17-g4386af6 I wondered what is the plan?

**Comment by effigies** [comment]
There are some open PRs, but I haven't found time to push on this in months. We can release, if what's in `master` is worth having without waiting on something in queue.

**Comment by aebrahim** [comment]
As a first time user of this project, I found the fixes from 7353832af2971eb6128cc513499cdef5fd12c099 as nice to have, if y'all are considering cutting a new release soon.

**Comment by effigies** [comment]
Yes, we should go ahead and release soon... I was hoping to get #238 and #256 in, but I don't really have the time to push on this right now, and we shouldn't hold up a release for nice-to-haves.

**Comment by effigies** [comment]
0.20 is released.
